stable/insights-agent/templates/goldilocks/rbac.yaml

```diff
@@ -16,7 +16,7 @@ rules:
   - apiGroups:
       - 'apps'
     resources:
-      - 'deployments'
+      - '*'
```

Comment:
@ivanfetch-fw we should just change this to bind to the `view` clusterrole

Reply:
Ah yes, that's nicer. I've kept the Goldilocks ClusterRole for the VPA access, and added a binding for view for everything else.